contrib/test_jenkins.sh

```diff
@@ -306,6 +306,30 @@ get_rdma_device_ip_addr() {
 	echo $ipaddr
 }
 
+get_tcp_ip_addr() {
+    if ! which ibdev2netdev >&/dev/null
+    then
+        return
+    fi
+
+    #get the interface of the ip address that is used to reach The Google Public DNS IP addresses (IPv4).
+    #making the assumption that the pure Ethernet IPv4 address on the host is the one used for this purpose.
+    eth_iface=$(ip route get 8.8.8.8 | awk -F 'dev' '/dev/{print $2}' | awk '{print $1}')
+    ipoib_roce_ifaces=$(ibdev2netdev)
```

Comment:
can we just find the default gateway?

Reply:
isn't this it?

Reply:
pls simplify the code

Reply:
https://www.unixmen.com/how-to-find-default-gateway-in-linux/#:~:text=You%20can%20find%20default%20gateway,netstat%20commands%20in%20Linux%20systems.&text=The%20above%20output%20shows%20my,and%20G%20stands%20for%20Gateway.

`ip route show|sed -n 's/default via \(\S*\) dev.*/\1/p'`